pages/api/users/index.js

```diff
@@ -13,7 +13,10 @@ export default async function handler(req, res) {
 
   await connectMongo();
 
-  const fileProfiles = findAllBasic();
+  let fileProfiles = findAllBasic();
+  const initialRandomUsers = req?.query?.initialRandomUsers == 'true';
```

Comment:
I don't think the `?` in `req?.` is required, also maybe use something like `const { initialRandomUsers } = req.query` and include `keywords` also